custom_components/tahoma/strings.json

```diff
@@ -20,20 +20,13 @@
   },
   "options": {
     "step": {
-      "init": {
-        "description": "Select temperature sensors for thermostats and preset temperatures.\nNote: reset temperatures do not apply to the Smart Thermostat.",
+      "update_interval": {
+        "title": "Update Interval",
+        "description": "The Somfy TaHoma integration periodically checks for new events. Change the update interval to a lower value if you use a physical remote to control your devices.",
```

Comment:
Could someone help me with the French translation? :-)

Reply:
I can help ;) But I don't understand your last sentence: 
>if you use a physical remote to control your devices.

I would suggest: 
> if you want more frequent update.

Reply:
>The Somfy TaHoma integration periodically retrieves new events. Change the update interval to a lower value if you if you want more frequent updates, for example when you also control your devices outside Home Assistant.

Good one, this is the latest text, does it make more sense? I think it would be good to mention when you need a higher update frequency, since it is not necessary if you only use Home Assistant.